Zhana_Konsulova_7MI0800248_group6/Prakt2_zad2/Prakt2_zad2/Prakt2_zad2.cpp

```diff
@@ -0,0 +1,90 @@
+// Prakt2_zad2.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+
+
+struct Pair
+{
+    int first, second;
+    void Read()
+    {
+        std::cin >> first >> second;
+    }
+    void Print() const
+    {
+        std::cout << "(" << first << ", " << second << ")" << std::endl;
+    }
+};
+
+bool IsFirstPairBigger(const Pair& p1,const Pair& p2)
+{
+    return
+        p1.first > p2.first ||
+        (p1.first == p2.first && p1.second > p2.second);
+}
+
+Pair TheBiggest(Pair arr[100])
+{
+    Pair theBiggest = arr[0];
+    for (int i = 1; i < 100; i++)
+    {
+        if (IsFirstPairBigger(arr[i], theBiggest))
+            theBiggest = arr[i];
+    }
+    return theBiggest;
+}
+
+
+
+void Swap(Pair& p1, Pair& p2)
+{
+    Pair temp = p1;
+    p1 = p2;
+    p2 = temp;
+}
+void SortTheArrOfPairs(Pair * arr, size_t size)
+{
+    for (int i = 0; i < size; i++)
+    {
+        bool isSwapped = false;
+        for (int j = 0; j < size - 1 - i; j++)
+        {
+            if (IsFirstPairBigger(arr[j], arr[j + 1]))
+            {
+                Swap(arr[j], arr[j + 1]);
+                isSwapped = true;
+            }
+        }
+
+        if (!isSwapped)
+            return;
+    }
+    
+}
+
+void Print(Pair* arr, size_t size)
+{
+    for (int i = 0; i < size; i++)
+        std::cout << "(" << arr[0].first << ", " << arr[1].second << ")" << std::endl;
```

Comment:
Guess this should be `std::cout << "(" << arr[i].first << ", " << arr[i].second << ")" << std::endl;` instead ?